lib/src/session.rs

```diff
@@ -47,6 +48,8 @@ pub struct Session {
     downstream_client_addr: SocketAddr,
     /// The IP address and port that received this session.
     downstream_server_addr: SocketAddr,
+    /// The amount of time we've spent on this session in ms.
+    pub active_cpu_time_us: Arc<AtomicU64>,
```

Comment:
```suggestion
    /// The amount of time we've spent on this session in microseconds.
    pub active_cpu_time_us: Arc<AtomicU64>,
```

Was reading this PR out of curiosity and to practice reading Rust and noticed these units mismatch.

Reply:
Yes, they do! Thanks for the catch!